classes/fields/datetime.php

```diff
@@ -495,24 +499,33 @@ public function format_date( $options, $js = false ) {
 		switch ( (string) pods_v( static::$type . '_type', $options, 'format', true ) ) {
 			case 'wp':
 				$format = get_option( 'date_format' );
+
 				if ( $js ) {
-					$format = $this->convert_format( $format, array( 'source' => 'php' ) );
+					$format = $this->convert_format( $format, array( 'source' => 'php', 'type' => 'date' ) );
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: d9c687563a9a084eb4636e63ac81a989</sub>